src/base/ClpMap.h

```diff
@@ -144,7 +144,6 @@ template <class Key, class Value, uint64_t MemoryUsedBy(const Value &)>
 void
 ClpMap<Key, Value, MemoryUsedBy>::setMemLimit(const uint64_t newLimit)
 {
-    assert(newLimit >= 0);
```

Comment:
It looks like the rebasing problems continue to plaque this branch -- this change does not belong to this PR IIRC.